pkg/server/server.go

```diff
@@ -28,13 +28,19 @@ func ListenAndServe(ctx context.Context, address string, config *config.Config)
 		schema.Store = release.New(config)
 		schema.CollectionMethods = []string{http.MethodGet}
 	})
-	apiroot.Register(server.Schemas, []string{"v1-release"}, nil)
 
+	switch pathPrefix {
+	case "":
+		pathPrefix = "-"
+	default:
+		pathPrefix = "-" + pathPrefix + "-"
```

Comment:
As per my suggested changes, this sanitization is no longer necessary but we should be sure to trim any trailing "/"